### PR TITLE
updating ci build and signing behavior

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -48,32 +38,29 @@ jobs:
 
   steps:
   - template: build/install-dotnet.yml
-  
-  - task: 1ESHostedPoolValidation@1
-  - pwsh: |
-      $packageSuffix = 0
-      if (-not $$(hasTag)) {
-        $packageSuffix = $(buildNumber)
-        Write-Host "No git tag found. Setting packages suffix to '$packageSuffix'"
```
Comment (soninaren) [inline azure-pipelines.yml:57]: I am guessing we don't use package suffix anymore.

**Review by soninaren — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (soninaren) [reply azure-pipelines.yml:57]: nevermind, i see we are directly using buildNumber

Comment (brettsam) [reply azure-pipelines.yml:57]: we do -- it's just added to the 'pack' step (`--version-suffix`)

```
      custom: 'pack'
      ${{ if eq(variables.hasTag, true) }}:
        arguments: '--no-build -c Release -o $(buildOutputDirectory)'
      ${{ else }}:
        arguments: '--no-build -c Release -o $(buildOutputDirectory) --version-suffix -$(buildNumber)'
```

Comment (soninaren) [reply azure-pipelines.yml:57]: yeah, i just saw that a little later.

**Pull request merged**